packages/functional-tests/scripts/start-services.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -ex
+
+# This routine was formerly part of in test-ci.sh. It has been
+# split up so it can be run in separate steps in the CI,
+# resulting in more meaningful timing metrics.
+
+DIR=$(dirname "$0")
+
+cd "$DIR/../../../"
+
+mkdir -p ~/.pm2/logs
+mkdir -p artifacts/tests
+CI=true yarn workspaces foreach \
+    --verbose \
+    --topological-dev \
+    --include 123done \
+    --include browserid-verifier \
+    --include fxa-auth-server \
+    --include fxa-content-server \
+    --include fxa-graphql-api \
+    --include fxa-payments-server \
+    --include fxa-profile-server \
+    --include fxa-settings \
+    run start > ~/.pm2/logs/startup.log
+
+# stop services that aren't needed. These are 'watching' services, and they just
```

Comment:
@ashrivastava-qa We make sure to stop these pm2 processes since they result in watchers, which are purely for development, and just waste memory in the CI. I also have suspicions that sometimes they kick in during tests and cause problems. At any rate disabling them should help some with memory usage, and hopefully flakiness.